client/app/directives/right-click.directive.js

```diff
@@ -14,14 +14,10 @@
 
         function link(scope, element, attrs) {
```

Comment:
@yegor86 
It is still used in the files directive:
mcc-right-click="$emit('rightClickOnFile', $event, item)"
But now we use angular's $emit to fire an event instead of calling rightClickOnItem in the controller. The event is handled in the directive. 
It is still easy to test and review and there are less code.